### PR TITLE
Redirect for @akosyakov

### PR DESCRIPTION
**Pull request by jankeromnes**
This is mostly a private joke. 😂 [Context](https://gitpod.slack.com/archives/C01KGM9DVRC/p1650626148227359) (internal)

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@jankeromnes: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/dashboard/src/App.tsx [hunk 1]
```diff
@@ -279,6 +279,12 @@ function App() {
         window.location.host = "www.gitpod.io";
         return <div></div>;
     }
+    
+    // redirect for Anton
+    if (isGitpodIo() && && window.location.pathname === "/akosyakov") {
```
Comment (jankeromnes) [inline components/dashboard/src/App.tsx:284]: TODO: Improve JavaScript skillz:

```suggestion
    if (isGitpodIo() && window.location.pathname === "/akosyakov") {
```

**Review by akosyakov — DISMISSED** [review]
I approve it 😆

**Comment by jankeromnes** [comment]
(Closing as this was mostly a joke, the build failed, and I don't expect a random approval from team WebApp to discreetly sneak this in 😝)

**Pull request closed without merging**